### PR TITLE
[DO NOT MERGE] ROX-10619: Demo code change workflow

### PR DESCRIPTION
**Pull request by msugakov**
## Description

See JIRA.

## Checklist
- [ ] Investigated and inspected CI test results
- ~~[ ] Unit test and regression tests added~~ not for readme update.
- ~~[ ] Evaluated and added CHANGELOG entry if required~~ not needed for this change.
- ~~[ ] Determined and documented upgrade steps~~ no deployment changes.
- ~~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~ not needed for this change.

## Testing Performed

* Relying on CI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 [![CircleCI][circleci-badge]][circleci-link]
 
+# Hello, I'm a readme change
```
Comment (msugakov) [inline README.md:3]: How about to make it more fancy?
```suggestion
# Hello, I'm a fancy readme change
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+
+Blah, some text here.
```
Comment (msugakov) [inline README.md:5]: Please add more text here.

Comment (msugakov) [reply README.md:5]: Sorry, no time now.

Comment (msugakov) [reply README.md:5]: Done. Please check.

**Review by vikin91 — CHANGES_REQUESTED** [review]
Sorry, the readme is not fancy enough.

**Pull request closed without merging**